.docker/scripts/create-test-data.sh

```diff
@@ -85,10 +86,10 @@ curl -LsH "Authorization: ${API_KEY}" \
 #
 
 # Creating test data hierarchy which creates organisations assigned to datasets
-ckan_cli create-test-data hierarchy
+ckan_cli seed hierarchy
```

Comment:
This is unnecessary since 'ckan_cli' converts it automatically when on CKAN 2.9. Hard-coding it to "seed" breaks CKAN 2.8 support without any benefit.

Reply:
Sorry, this should be here, I've been using it locally.

Reply:
Yes, it will still work with this change, so long as you're on CKAN 2.9. However, it should still work fine without this change, and it will then be compatible with CKAN 2.8.